infra/prover/tomls/bonsai/broker-prod-11155111.toml

```diff
@@ -10,8 +10,8 @@ max_mcycle_limit = 8_000
 max_concurrent_locks = 3
 balance_warn_threshold = "5"
 balance_error_threshold = "1"
-stake_balance_warn_threshold = "5"
-stake_balance_error_threshold = "1"
+stake_balance_warn_threshold = "15"
+stake_balance_error_threshold = "10"
```

Comment:
these might be too high though, as the USDC faucet can only issues 10 per hour, meaning it might be annoying for the brokers at the beginning at least